CIP-1852/CIP-1852.md

```diff
@@ -0,0 +1,61 @@
+---
+CIP: 1852
+Title: HD (Hierarchy for Deterministic) Wallets for Cardano
+Authors: Sebastien Guillemot <sebastien@emurgo.io>, Matthias Benkort <matthias.benkort@iohk.io>
+Comments-URI: https://forum.cardano.org/t/cip1852-hd-wallets-for-cardano/41740
+Status: Draft
+Type: Standards
+Created: 2019-10-28
+License: CC-BY-4.0
+---
+
+## Abstract
+
+Cardano extends the [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) by adding new chains used for different purposes. This document outlines how key derivation is done and acts as a registry for different chains used by Cardano wallets.
+
+## Terminology
+
+### Derivation style
+
+Cardano does not use [BIP32](https://github.com/bitcoin/bips/blob/master/bip-0032.mediawiki) but actually uses [BIP32-Ed25519](https://cardanolaunch.com/assets/Ed25519_BIP.pdf). The `-Ed25519` suffix is often dropped in practice (ex: we say the Byron release of Cardano supports [BIP44](https://github.com/bitcoin/bips/blob/master/bip-0044.mediawiki) but in reality this is BIP44-Ed25519).
```

Comment:
https://cardanolaunch.com/assets/Ed25519_BIP.pdf is a lost page - do we have a different link we can use? 
Should we use https://ieeexplore.ieee.org/document/7966967 ?

Reply:
I don't know if there's anywhere to get the PDF without having to pay or creating an account. The old website was hosted by the CF I think, so maybe they have the PDF hosted on a different URL now? Maybe it can be uploaded as part of the IOHK library instead?